### ISSUE TITLE
Confirm legacy `bibxml-nist` data set items covered by `relaton-data-nist`, produce map

### ISSUE THREAD

**Issue by ronaldtse** [ietf-ribose/bibxml-project#20]
We also need to build a map between the old filenames to the new identifiers.

**Comment by ronaldtse** [comment]
@rjsparks could I confirm that we need to map the legacy paths from bibxml-nist? This dataset is not listed in the official xml2rfc.tools.ietf.org website. Thanks.

**Comment by rjsparks** [comment]
Yes - the text listing on that website hasn't caught up with the development of the datasets - the nist dataset is fairly new and is being served at, e.g., https://xml2rfc.tools.ietf.org/public/rfc/bibxml-nist/reference.NIST.AMS.100-1.xml

**Comment by ronaldtse** [comment]
@rjsparks got it, then we will need to provide a one to one mapping.

@mico could you please help implement a script that automatically generates a map between the two datasets?

**Comment by ronaldtse** [comment]
@rjsparks is there a way to rsync fetch the bibxml-nist? The rsync instructions on the xml2rfc.tools.ietf.org page doesn't provide that dataset:

```
rsync -avuz xml2rfc.tools.ietf.org::xml2rfc.bibxml
```

**Comment by rjsparks** [comment]
I've added bibxml-nist/ and bibxml-nist-new/ to that rsync point (it doesn't have a bibxml{number} alias). bibxml-nist-new is the results of an improved implementation Tony Hansen was working on.
Please also review the RFPs discussion of this dataset.

**Comment by ronaldtse** [comment]
@rjsparks Thanks. I re-read the RFP on the portion on bibxml-nist. It mentions bibxml-nist but not what happens with bibxml-nist-new.

Is `bibxml-nist-new` being used by the community? If not, we probably shouldn't build a legacy map for it. Its source data and that of the new data set is identical (the NIST Tech Pubs collection, available on GitHub: https://github.com/usnistgov/NIST-Tech-Pubs/tree/nist-pages/xml) (relevant code: (defined here: https://svn.ietf.org/svn/tools/xml2rfc/website/rfcs/bibxml/bibxml-nist/gen-bibxml-nist))

**Comment by rjsparks** [comment]
No - it is not in widespread use.

**Comment by ronaldtse** [comment]
@rjsparks thanks. In this case I propose we only map `bibxml-nist` and ignore `bibxml-nist-new`. We're using the same data source as that of `bibxml-nist-new`, so we're not going to lose anything. Thanks.

**Comment by ronaldtse** [comment]
We will need to produce a mapping between the legacy dataset of [bibxml-nist](http://xml2rfc.tools.ietf.org/public/rfc/bibxml-nist/) to the new NIST-PubID-named dataset of [relaton-data-nist](https://github.com/ietf-ribose/relaton-data-nist).

(filename to filename)

**Comment by ronaldtse** [comment]
@mico can you take on the mapping task since you are familiar with NIST PubID? Thanks.

**Comment by ronaldtse** [comment]
@mico you will need to write a short script that ensures every entry of https://github.com/ietf-ribose/bibxml-data-archive/tree/main/bibxml-nist-new is available in https://github.com/ietf-ribose/relaton-data-nist/tree/main/data
, and provide a YAML mapping file. Thanks!

**Comment by mico** [comment]
@ronaldtse what the output I should generate?
Old filename to new filename?
What if file missing?

**Comment by mico** [comment]
script produces mapping here: https://github.com/ietf-ribose/bibxml-to-relaton-mapping
